css/ui/style.css

```diff
@@ -124,19 +119,6 @@
     overflow-x: hidden;
 }
 
-.scratchpad-ace-editor .ace_sb {
-    overflow-y: auto;
-}
-
-.scratchpad-ace-editor .ace_active_line {
```

Comment:
"ace_active_line" no longer seems to select anything, and I couldn't find an equivalent class. I couldn't find the string in the ace codebase either.

Reply:
Update: I found the string, it's ace_active-line.